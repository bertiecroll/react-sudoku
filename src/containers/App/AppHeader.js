import React from 'react'
import logo from 'images/logo.svg'

export function AppHeader () {
  return (
    <header className="AppHeader">
      <img src={logo} className="App-logo" alt="logo" />
      <h2 className="App-title">React Sudoku</h2>
    </header>
  )
}
