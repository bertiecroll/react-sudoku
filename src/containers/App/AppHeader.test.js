import React from 'react';
import { shallow } from 'enzyme'
import { AppHeader } from './AppHeader';

describe('AppHeader', () => {
  it('renders component content', () => {
    const wrapper = shallow(<AppHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
