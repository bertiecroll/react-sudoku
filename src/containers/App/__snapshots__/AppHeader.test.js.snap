// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders component content 1`] = `
<header
  className="AppHeader"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h2
    className="App-title"
  >
    React Sudoku
  </h2>
</header>
`;
